openapi.go

```diff
@@ -171,17 +173,13 @@ func RegisterOpenAPIOperation[T, B any](s *Server, route Route[T, B]) (*openapi3
 
 	// Request Body
 	if route.Operation.RequestBody == nil {
-		bodyTag := schemaTagFromType(s, *new(B))
+		bodyTag := SchemaTagFromType(s, *new(B))
 
-		if bodyTag.name != "unknown-interface" {
-			requestBody := newRequestBody[B](bodyTag, []string{"application/json", "application/xml"})
-			s.OpenApiSpec.Components.RequestBodies[bodyTag.name] = &openapi3.RequestBodyRef{
-				Value: requestBody,
-			}
+		if bodyTag.Name != "unknown-interface" {
+			requestBody := newRequestBody[B](bodyTag, route.AcceptedContentTypes)
 
 			// add request body to operation
 			route.Operation.RequestBody = &openapi3.RequestBodyRef{
-				Ref:   "#/components/requestBodies/" + bodyTag.name,
```

Comment:
So are we just electing to not use the references moving foward?

Reply:
Simply not the requestBody references, but still other references.

Note that it's just because I want to ship Variadic route options, it's been a month and I want to move forward. We'll probably reuse requestBodies refs later but I need to think about a good way to do so.

Reply:
That is fair.

I think I may give a proof concept after I add the global request type feature. 

Cheers!